index.md

```diff
@@ -1,131 +1,198 @@
+Instead we should look up to people who help their communities, people who can effectively lead and people who serve. 
+We need to look up to people who work to ensure everyone has their basic needs covered. We need to look up to people 
+who can inspire future generations.
 
 
-Treat people with care and respect
-----------------------------------
+Treat people equally and with respect
+-------------------------------------
 
-We must always treat people with the care and respect with which we wish to be treated. All workers should be treated and paid fairly and we need to recognize that our essential workers who saved us during the pandemic were the most poorly paid people. We must fix this and pay everyone a fair living wage and provide safe working conditions.
+We must always treat people equally with care and respect with which we wish to be treated. All workers and teachers 
+should be treated and paid fairly and we need to recognize that our essential workers who saved us during the 
+pandemic were the most poorly paid people. We must fix this and pay everyone a fair living wage and provide safe 
+working conditions.
 
-We should also be excellent to others -- everyone is allowed to have a bad day once in a while, but this is no reason to disrespect others. Give people as much space as you can, but be as polite and fair as you can be.
+We should also be excellent to others -- everyone is allowed to have a bad day once in a while, but this is no reason 
+to disrespect others. Give people as much space as you can, but be as polite and fair as you can be.
 
 
-Demand Transparency
--------------------
+Demand transparency and facts
+-----------------------------
 
-The epidemic showed that transparent governments had the best response to the virus. Sadly very few governments are transparent since transparency makes it more difficult to hide bad deeds. Many governments will claim that transparency is not possible for national security reasons, when only a small amount of information really must remain classified.
+The epidemic showed that transparent governments had the best response to the virus. Sadly very few governments are 
+transparent since transparency makes it more difficult to hide bad deeds. Many governments will claim that transparency 
+is not possible for national security reasons, when only a small amount of information really must remain classified.
 
-In our competitive world we default to data being closed and not sharing with anyone. 
-Once again we should examine the open source and open data movements to see how free sharing of information can drastically improve life for everyone.
+In our competitive world we default to data being closed and not sharing with anyone.  Once again we should examine the 
+open source and open data movements to see how free sharing of information can drastically improve life for everyone.
+We must learn to be more open to allow people to engage with their governments for freely.
```

Comment:
"to engage with their governments for freely." ? I don't understand this phrase, is there an error or it's me?